src/app/(app)/page.client.tsx

```
@@ -148,7 +148,11 @@ export function AppPageClient({
     };
   }, [invoiceDataState?.qrCodeData, invoiceDataState?.qrCodeIsVisible]);
 
-  useShowRandomCTAToastOnIdle();
+  // Only show CTA toast on idle in non-CI environments
+  if (!process.env.CI) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useShowRandomCTAToastOnIdle();
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)